site/pages/api/add-consequence.ts

```diff
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { COOKIES_ADD_CONSEQUENCE_INFO, COOKIES_ADD_CONSEQUENCE_ERRORS } from "../../constants/index";
+import { ErrorInfo } from "../../interfaces";
+import { redirectTo, setCookieOnResponseObject } from "../../utils/apiUtils";
+import { AddConsequenceProps, ConsequenceType, TransportMode } from "../add-consequence";
+
+const addConsequence = (req: NextApiRequest, res: NextApiResponse): void => {
+    const errors: ErrorInfo[] = [];
+
+    const formFields: AddConsequenceProps = req.body as AddConsequenceProps;
```

Comment:
Instead of doing this could be change req: NextApiRequest on line 7 to be an interface of req: NextApiRequestWithConsequences

Reply:
done